.github/workflows/regressions.yml

```diff
@@ -59,6 +59,7 @@ jobs:
                             repository: tokio-rs/tokio
                           - name: amethyst
                             repository: amethyst/amethyst
+                            analyzeDependencies: false
```

Comment:
I excluded dependencies in `amethyst` because [this assertion](https://github.com/intellij-rust/intellij-rust/blob/79e84f4b69a37f00eae854201ca5cb5893a37eab/src/main/kotlin/org/rust/lang/core/resolve/ImplLookup.kt#L462) fails there, but it doesn't reproduce in IDE for some reason

Reply:
Created #9587